### DISCUSSION TITLE
Developer meeting (October 4th) - Discussion on how to handle the increasing complexity of the expansion decks

### DISCUSSION THREAD

**Discussion by evoggy** [bitcraze/discussions#931] [Announcements]
The next developer meeting will be on the 4th of October! In this meeting we will be presenting some of the issues we're seeing with the increasing complexity of the expansion decks moving forward. We're also going to be presenting some of the possible solutions we've been discussing around the office and we're hoping to get some feedback from the community.

Date: Wednesday, October 4th 2023
Time: 3 PM Central European Summer Time ([check your timezone](https://www.worldtimebuddy.com/?qm=1&lid=5128581,100,2692969,12,14,1816670&h=2692969&date=2023-10-04&sln=15-16&hf=1))
Topic: Discussion on how to handle the increasing complexity of the expansion decks
Host: @evoggy 
Meeting link: Meeting is over! Find video [here](https://youtu.be/P17C6305E1w)
Agenda:
15:00 - 15:30: Presentation and discussion
15:30 - 16:00: Generic support questions

Indicate with 👍 if you are planning to join!

**Comment by slivingston** [comment]
I missed this. Did you record a video, or are the slides available?

**Reply by krichardsson** [reply]
You can find [the video here](https://youtu.be/P17C6305E1w)
